crates/hardforks/src/ethereum/mainnet.rs

```diff
@@ -0,0 +1,67 @@
+//! Ethereum Mainnet hardfork starting points
+
+/// Homestead hard fork activation block
+pub const MAINNET_HOMESTEAD_BLOCK: u64 = 1_150_000;
```

Comment:
```suggestion
/// Frontier hard fork activation block
pub const MAINNET_FRONTIER_BLOCK: u64 = 0;
/// Homestead hard fork activation block
pub const MAINNET_HOMESTEAD_BLOCK: u64 = 1_150_000;
```
let's not assume it is common knowledge that the frontier block is the genesis block and include it as constant